modules/items/tests/e2e/items.e2e.tests.js

```diff
@@ -1,10 +1,129 @@
 'use strict';
 
 describe('Items E2E Tests:', function () {
+  var admin = {
+    username: 'seedadmintest',
+    password: 'P@$$w0rd!!'
+  };
+  var testItem = {
+  	title: 'title',
+  	module: 'thermodynamics',
+  	category: 'beaker',
+  	count: '5',
+  	content: 'yep'
+  };
+  var category = {
+  	title: 'title',
+  	hexColor: '#FAE123'
+  };
+  var module = {
+  	title: 'title',
+  	hexColor: '#123FAE'
+  };
+
+  var signout = function () {
+    // Make sure user is signed out first
+    browser.get('http://localhost:3001/signout');
+    // Delete all cookies
+    browser.driver.manage().deleteAllCookies();
+  };
+
+  var signin = function() {
+    signout();
+    browser.get('http://localhost:3001/signin');
+      // Enter UserName
+      element(by.model('vm.credentials.usernameOrEmail')).sendKeys(admin.username);
+      // Enter Password
+      element(by.model('vm.credentials.password')).sendKeys(admin.password);
+      // Click Submit button
+      element(by.css('button[type="submit"]')).click();
+  };
   describe('Test items page', function () {
-    it('Should report missing credentials', function () {
-      browser.get('http://localhost:3001/items');
-      expect(element.all(by.repeater('item in items')).count()).toEqual(0);
+    signin();
```

Comment:
put this in a before(function() {...}); so it does this before it runs the tests